src/components/main.jsx

```diff
@@ -122,6 +127,22 @@ export default class Main extends React.Component {
     if (this.state.tool === 'measure') {
       this.setState({ lengthMeasurePoints: [ ...this.state.lengthMeasurePoints, { lat, lon } ] });
     }
+    if(this.state.tool == 'route-planner'){
```

Comment:
rozumiem ze by bolo super keby sme vsetci pisali syntax rovnako, a keby toto bol jediny projekt ktory programujem, tak s tebou suhlasim. 

ale vzhladom na to v kolkych inych programovacich jazykoch este aktivne robim (typescript, coffesscript, js, ruby) a na kolkych roznych projektoch, snazim sa pouzivam minimalisticku syntax z kazdeho jazyka, inak by som mal problemy sa rychlo prepinat medzi jazykmi/projektami.

jedine co je dobre je pouzivat rovnaky sposob odsadzovania, ostatne je pre mna strata casu.